.github/workflows/code_health.yaml

```
@@ -37,22 +37,47 @@ jobs:
                     exit 1
                   fi
     prettier:
-        name: Prettier Check
         runs-on: ubuntu-latest
         steps:
             - uses: GitHubSecurityLab/actions-permissions/monitor@v1
               with:
                   config: ${{ vars.PERMISSIONS_CONFIG }}
             - name: Checkout Repository
               uses: actions/checkout@v2
-            - name: Run Prettier
-              id: prettier-run
-              uses: rutajdash/prettier-cli-action@d42c4325a3b344f3bd4be482bc34de521998d557
+            - uses: actions/setup-node@v4
               with:
-                  config_path: ./.prettierrc.yml
-            - name: Prettier Output
-              if: ${{ failure() }}
-              shell: bash
+                  node-version-file: package.json
+            - name: install dependencies
+              run: |
+                  npm ci
+            - run: |
+                  npm run reformat
+            - name: Check for uncommitted files
               run: |
-                  echo "The following files are not formatted:"
-                  echo "${{steps.prettier-run.outputs.prettier_output}}"
+                  export FILES=
+                  FILES=$(git ls-files -o -m --directory --exclude-standard --no-empty-directory)
+                  export LINES=
+                  LINES=$(echo "$FILES" | awk 'NF' | wc -l)
+                  if [ "$LINES" -ne 0 ]; then
+                    echo "Detected files that need to be committed:"
+                    echo "${FILES}"
+                    echo ""
+                    echo "Try running: npm run reformat"
+                    exit 1
+                  fi
+    eslint:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: GitHubSecurityLab/actions-permissions/monitor@v1
+              with:
+                  config: ${{ vars.PERMISSIONS_CONFIG }}
+            - name: Checkout Repository
+              uses: actions/checkout@v4
+            - uses: actions/setup-node@v4
+              with:
+                  node-version-file: package.json
```

```suggestion
                  node-version-file: package.json
                  cache: "npm"
```
would also be good to add to the steps above